src/Type/Doctrine/Query/QueryResultTypeWalker.php

```diff
@@ -812,43 +844,39 @@ public function walkSelectExpression($selectExpression)
 			$resultAlias = $selectExpression->fieldIdentificationVariable ?? $this->scalarResultCounter++;
 			$type = $this->unmarshalType($expr->dispatch($this));
 
-			if (class_exists(TypedExpression::class) && $expr instanceof TypedExpression) {
```

Comment:
This was dead block of code as `class_exists(TypedExpression::class)` is always false.

Reply:
https://github.com/phpstan/phpstan-src/pull/2838